src/test/resources/CucumberSmokeTest.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="CucumberSmokeTest" >
+
+    <test name="CucumberLoginTest" preserve-order="false">
+        <parameter name="BaseURL" value="https://bbb.testpro.io/" />
+        <classes>
+            <class name="CucumberRunner"/>
+        </classes>
+    </test>
+</suite>
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/225712844-bec77dca-18cd-4f28-b1c2-1e6e929fb9e4.png)

Hint

`src/test/resources/features`

![image](https://user-images.githubusercontent.com/104761482/225714943-07fc5b18-b0e6-4fbf-9c74-4f4e8e9cba59.png)